back/lib/omniauth_methods/base.rb

```diff
@@ -24,12 +24,11 @@ def filter_auth_to_persist(auth)
 
     # @return [Array<Symbol>] Returns a list of user attributes that can be updated from the auth response hash
     def updateable_user_attrs
-      []
-    end
-
-    # @return [Boolean] If existing user attributes should be overwritten
-    def overwrite_user_attrs?
-      true
+      result = []
+      # If password_login is disabled, users cannot update their emails on UI,
+      # but we still want to keep their emails up to date.
+      result << :email if !AppConfiguration.instance.feature_activated?('password_login')
+      result
```

Comment:
```suggestion
      [].tap do |attrs|
        # If password_login is disabled, users cannot update their emails on UI,
        # but we still want to keep their emails up to date.
        attrs << :email if !AppConfiguration.instance.feature_activated?('password_login')
      end
```

Reply:
I slightly prefer my version because it doesn't introduce indents :slightly_smiling_face: